code/__HELPERS/areas.dm

```diff
@@ -141,6 +185,15 @@ GLOBAL_LIST_INIT(typecache_powerfailure_safe_areas, typecacheof(/area/station/en
 	SEND_GLOBAL_SIGNAL(COMSIG_AREA_CREATED, newA, oldA, creator)
 	to_chat(creator, span_notice("You have created a new area, named [newA.name]. It is now weather proof, and constructing an APC will allow it to be powered."))
 	creator.log_message("created a new area: [AREACOORD(creator)] (previously \"[oldA.name]\")", LOG_GAME)
+
+	//check areas of turfs merged into the new area if their empty
+	for(var/area_name in collected_areas)
+		var/area/merged_area = collected_areas[area_name]
+		//no more turfs in this area. Time to clean up
+		if(!merged_area.has_contained_turfs())
+			message_admins("Area [area_name] has been deleted!.")
```

Comment:
```suggestion
			message_admins("Area [area_name] has been deleted!")
```

Does this need to be a message admins, anyways? If this is a failstate, would it be apt to have it be a stack trace instead?

Reply:
why do we need to tell anyone?

Reply:
> why do we need to tell anyone?

Fair enough. I'll remove it completely

Reply:
👍